foundationsync/indico_foundationsync/sync.py

```diff
@@ -46,6 +41,15 @@ def OutputTypeHandler(cursor, name, defaultType, size, precision, scale):
         return cursor.var(unicode, size, cursor.arraysize)
 
 
+def _get_room_role_map(connection, logger):
+    roles = defaultdict(set)
+    cursor = connection.cursor()
+    cursor.execute('SELECT BUILDING, FLOOR, ROOM_NUMBER, EMAIL FROM aispub.app_indico_space_managers')
+    for row in cursor:
+        roles[row[:3]].add(row[3])
```

Comment:
`row['EMAIL']`?

i don't think using positional indexes for query results with more than one row is great, but i have to admit that it's convenient for the first 3 rows ;)

Reply:
Since I enumerate them above, I don't see that as a big deal. But yeah, `EMAIL` is better written as text.